assets/js/components/data-table.js

```diff
@@ -91,7 +92,10 @@ export const getDataTableFromData = ( data, headers, options ) => {
 	} );
 
 	return (
-		<div className={ `googlesitekit-table${ ( options && options.disableListMode ? '' : ' googlesitekit-table--with-list' ) }` }>
+		<div className={ classnames(
+			'googlesitekit-table',
+			{ 'googlesitekit-table--with-list': options && ! options.disableListMode }
```

Comment:
That's not exactly what this conditional was. Previously if `! options === true`  this would render `'googlesitekit-table--with-list'`. So this conditional should be: `! options || ! options.disableListMode`